pkg/cmd/cli/cmd/tag.go

```diff
@@ -419,6 +421,7 @@ func (o TagOptions) Run() error {
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      destNameAndTag,
 					Namespace: o.destNamespace[i],
+					Labels:    o.labels,
```

Comment:
This might be an API backwards compatibility break.  Old clients would be broken.